arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -1006,11 +1028,17 @@
           }
         ],
         "responses": {
+          "200": {
```

Comment:
Is the service being changed to now return 200 in cases where it previously did not?